### PR TITLE
NodeJS Quickstart: Logout from Auth0 authorization server

### PR DESCRIPTION
**Pull request by alexisluque**
Clear session at Auth0 authorization server calling `/v2/logout` endpoint when perform logout.
Sample also updated at: auth0-samples/auth0-nodejs-webapp-sample#44.

### CODE REVIEW

File: articles/quickstart/webapp/nodejs/01-login.md [hunk 1]
```diff
@@ -142,9 +144,9 @@ passport.deserializeUser(function (user, done) {
 In this example, following routes are implemented:
 
 * `/login` triggers the authentication by calling Passport's `authenticate` method. The user is then redirected to the tenant login page hosted by Auth0.
-* `/callback`is the route the user is returned to by Auth0 after authenticating. It redirects the user to the profile page (`/user`).
+* `/callback` is the route the user is returned to by Auth0 after authenticating. It redirects the user to the profile page (`/user`).
 * `/user` displays the user's profile.
-* `/logout` closes the local user session and redirects the user again to the root index `/`.
+* `/logout` closes the local user session, logout the user at authorization server and redirects the user again to the root index `/`.
```
Comment (joshcanhelp) [inline articles/quickstart/webapp/nodejs/01-login.md]: > closes ... logout ... redirects

Maybe "logs the user out of Auth0" or thereabouts?

File: articles/quickstart/webapp/nodejs/01-login.md [hunk 2]
```diff
@@ -189,14 +197,30 @@ router.get('/callback', function (req, res, next) {
...
+
   res.redirect('/');
 });
 
 module.exports = router;
 ```
 
 :::note
-This tutorial implements logout by closing the local user session. After logging out, the user's session in the Auth0 authentication server is still open. For other implementations, please refer to the [logout documentation](/logout). 
+This tutorial implements logout by closing the local user session and the user's session in the Auth0 authentication server as well. For other implementations, please refer to the [logout documentation](/logout).
```
Comment (joshcanhelp) [inline articles/quickstart/webapp/nodejs/01-login.md]: > ... and the user's session in the Auth0 authentication server

Consider:

> and the user's Auth0 session

Comment (joshcanhelp) [reply articles/quickstart/webapp/nodejs/01-login.md]: Also: 

https://auth0.com/docs/logout/

I think that's what this relative link is translated to ... if so, that's a 404

**Review by joshcanhelp — CHANGES_REQUESTED** [review]

File: articles/quickstart/webapp/nodejs/01-login.md (hunk 2 above)
Comment (albertoperdomo) [reply articles/quickstart/webapp/nodejs/01-login.md]: There's an issue with trailing slashes. https://auth0.com/docs/logout works.

File: articles/quickstart/webapp/nodejs/01-login.md [hunk 3]
```diff
@@ -189,14 +197,30 @@ router.get('/callback', function (req, res, next) {
 // Perform session logout and redirect to homepage
 router.get('/logout', (req, res) => {
   req.logout();
+
+  var returnTo = util.format(
```
Comment (joshcanhelp) [inline articles/quickstart/webapp/nodejs/01-login.md]: Need to move over the changes from the sample RE: port

**Review by joshcanhelp — CHANGES_REQUESTED** [review]

**Review by joshcanhelp — APPROVED** [review]

**Pull request merged**